__tests__/components/Markdown.spec.js

```diff
@@ -0,0 +1,275 @@
+/* globals expect, describe, it */
+import { shallow } from 'enzyme'
+import React from 'react'
+import renderer from 'react-test-renderer'
+import {
+  Provider
+} from '../../dist'
+import {
+  Markdown
+} from '../../dist/extra/Markdown'
+
+describe('Markdown component', () => {
```

Comment:
What was the reason to add markdown tests of our own? `marked` is much better tested and it feels redundant for us to also test the same things.

Reply:
@sradevski In short, its the same reasons we test anything.
Different combinations of sanitizers and marked options can lead to different results and rendering outputs. Additionally, by testing the output at the rendition level we can ensure that we maintain expected behaviour and a consistent API, even if we change the underlying implementation.